src/Colors/index.d.ts

```diff
@@ -9,8 +9,6 @@ export type ColorTransform = {
   get(): number;
 };
 
-export type Colors = {
-  (colorName: string): ColorTransform;
-} & {
-  [colorName: string]: ColorTransform;
-};
+declare const Color: (colorName: string) => ColorTransform;
```

Comment:
Can be Colors? instead of Color

Reply:
Since it is a default export, it can be. However, `Color` is more in line with the code.

https://github.com/rdkcentral/Lightning-SDK/blob/888ce77cf6275d6b027fe8f442e6abff046bf390/src/Colors/index.js#L105